src/core/lib/gprpp/arena.cc

```diff
@@ -48,7 +48,7 @@ Arena::~Arena() {
   }
 }
 
-Arena* Arena::Create(size_t initial_size) {
+static void* ArenaStorage(size_t initial_size) {
```

Comment:
Please use an anonymous namespace instead of `static`.

Reply:
Done.